### PR TITLE
Added  snackbar/Toast

### PR DESCRIPTION
**Pull request by haarsh24**

### CODE REVIEW

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.snackbar-container {
+  display: flex;
```
Comment (xii-vi) [inline components/snackbar.css]: if you are using flex everywhere define a common class for it.

**Review by xii-vi — APPROVED** [review]
Work on suggested comment, after that you can merge.

File: components/components-style.css [hunk 2]
```diff
@@ -6,6 +6,10 @@
 @import url("./images.css");
 @import url("./inputs.css");
 
+
+
+
+@import url("./snackbar.css");
 * {
   font-family: "Readex Pro", sans-serif;
   transition: all ease 0.5s;
```
Comment (devdeadviz) [inline components/components-style.css:15]: Add this universal style to reset the stylesheet and then import it where ever you need it.

File: components/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.snackbar-container {
+  flex-direction: row;
+  justify-content: center;
+}
+.snackbars {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
```
Comment (devdeadviz) [inline components/snackbar.css:9]: Just a suggestion, you can create separate layout styles for this.

File: components/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  background-color: var;
+  border-radius: 1rem;
+  height: 63vh;
+  border: 0.5rem solid var(--primary-color);
+  border-radius: 1rem;
+  margin: 1rem;
+  padding: 1rem;
+}
```
Comment (devdeadviz) [inline components/snackbar.css:17]: Just a suggestion, you can create separate styles for margins and paddings.

File: components/snackbar.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  padding: 0 0.5rem;
+  border-radius: 1rem;
+  margin: 1rem;
+}
+.baseline {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  display: none;
```
Comment (devdeadviz) [inline components/snackbar.css:38]: same here

**Review by devdeadviz — COMMENTED** [review]
Added some suggested changes, have a look at it.

**Pull request merged**